docs/gateway-api/gateway-api-v1.md

```diff
@@ -159,8 +159,8 @@ In AKO Gateway API Implementation, Gateway objects corresponds to following AVI
   3. Every `Secret` created corresponds to an `SSLKeyAndCertificate` object.
   4. `Addresses` in a Gateway specification gets added as static ip for `Vsvip` for parent VS.
   5. Each `hostname`, except `wild card hostnames`, specified in a Gateway listener is mapped to the `DNS` in the `Vsvip` of the parent VS.
-  6. `hostname` specified in HTTPRoute gets mapped to `VHMatch` in the childVS.
-  7. Every `Rule` in `HTTPRoute` corresponds to an `EVH Child Virtual Service`, with `Match` translated to `VH match` and `Filters` translated to `HTTPPolicySet` configuration. If no matches are specified for a particular HTTPRoute rule, a childVS with path as `/` in `VHmatch` will be created and attached to the parentVS corresponding to that rule.
+  6. Every `Rule` in `HTTPRoute` corresponds to an `EVH Child Virtual Service`, with `Match` translated to `VH match` and `Filters` translated to `HTTPPolicySet` configuration. If no matches are specified for a particular HTTPRoute rule, a childVS with path as `/` in `VHmatch` will be created and attached to the parentVS corresponding to that rule.
+  7. `hostname` specified in HTTPRoute gets mapped to `hosts` in the childVS.
```

Comment:
`childVS` --> `child VS`

Reply:
hostname mentioned in httproute, if it is complete fqdn will be part of VSVIP dnsinfo. 
all hostnames, mentioned in HTTPRoute, will be part of hostname of VHMatch of Child VS